### ISSUE TITLE
Aarlo doorbell integration

### ISSUE THREAD

**Issue by SohamJa1n** [twrecked/hass-aarlo#816]
Aarlo is running good so far apart from getting the glance card working but I was wondering if it was possible to use aarlo to create an automation so that when my arlo video doorbell rang, it would take over the homeassistant dashboard or at least show a pop-up preview to see who is at the door.

**Comment by twrecked** [comment]
It could be, I wouldn't know how to do it to be honest. You can definitely get a stream from the camera I'm just not sure how you'd go about displaying it.

**Comment by SohamJa1n** [comment]
Yeah I managed to set it up, I installed browser mod and then used that to create a popup when the doorbell rang with the aarlo glance card.

However it has a massive delay so doesn't actually show anyone at the door. Would saving a snapshot and then showing it work better?
